.travis.yml

```diff
@@ -1,19 +1,43 @@
-sudo: false
 language: python
 python:
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
+  - 3.6
 env:
-  - DJANGO="1.11,<2.0"
-  - DJANGO="2.0,<3.0"
+  - DJANGO=1.7
+  - DJANGO=1.8
+  - DJANGO=1.9
+  - DJANGO=1.10
+  - DJANGO=1.11
+  - DJANGO=2.0
+matrix:
+  exclude:
+    # Python/Django combinations that aren't officially supported
+    - { python: 3.5, env: DJANGO=1.7 }
+    - { python: 3.6, env: DJANGO=1.7 }
+    - { python: 3.6, env: DJANGO=1.8 }
+    - { python: 3.3, env: DJANGO=1.9 }
+    - { python: 3.6, env: DJANGO=1.9 }
+    - { python: 3.3, env: DJANGO=1.10 }
+    - { python: 3.6, env: DJANGO=1.10 }
+    - { python: 3.3, env: DJANGO=1.11 }
+    - { python: 2.7, env: DJANGO=2.0 }
```

Comment:
If we don't test with older Django versions then you can "include" python 2.7 and Django 1.11 instead.